### ISSUE TITLE
result to_dataframe not working

### ISSUE THREAD

**Issue by cimarie** [hakobera/luigi-bigquery#15]
``` python
def to_dataframe(self):
    import pandas as pd
    return pd.DataFrame(iter(self), columns=self._columns())
```

Use of this method inside my program returns error: "data argument can't be an iterator" (Python 2.7)
I fixed this locally by changing the code from `client.py` to:

``` python
def to_dataframe(self):
    import pandas as pd
    df = pd.DataFrame(columns=self._columns())
    i = 0
    for row in self._rows():
        df.loc[i] = row
        i += 1
    return df
```

**Comment by pedrocicoleme** [comment]
I was having this problem too. Fix in cimarie's post worked here.

**Comment by hakobera** [comment]
@cimarie @pedrocicoleme This issue is fixed by #17

**Comment by pedrocicoleme** [comment]
Thanks for the feedback @hakobera
